apps/myaccount/src/components/shared/header.tsx

```diff
@@ -207,6 +208,17 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
                     } }
                 />
             ) }
+            extensions={ [
+                {
+                    component: (
+                        <AppSwitch
+                            bottomMargin = { true }
```

Comment:
Fix other places as well.
```suggestion
                            bottomMargin={ true }
```